### DISCUSSION TITLE
Smile + Tablesaw Random Forest and K-Means

### DISCUSSION THREAD

**Discussion by jbsooter** [jtablesaw/tablesaw#1076] [Ideas]
@lwhite1 We are planning on creating tutorials for using Smile and Tablesaw together to perform Random Forest and K-means analysis for a Java course. These will be styled after the Moneyball tutorial. We would like to contribute them back to add to the Data Science section of the Tablesaw user guide. Is there any specific content that you would like to see included?

**Comment by lwhite1** [comment]
@jbsooter That sounds great.  As long as you have a good example it will be awesome. 

Let me know if you run into any issues.

**Reply by jbsooter** [reply]
@lwhite1 I am noticing that Github pages does not support my absolute filepaths (all links and images work fine when you look in the tablesaw repo). 

Pages does support the relative references that I left in. What is your preference on how to provide references in a Pages compatible way?

**Reply by lwhite1** [reply]
@jbsooter I looked at the URLS. The ones you used for Random Forest look like the first one below. The ones from the Moneyball tutorial are like the second one.

https://jtablesaw.github.io/docs/userguide/images/ml/random_forest/Decision_Tree_8.1.jpg
https://jtablesaw.github.io/tablesaw/userguide/images/ml/regression/moneyball-3_1.png

instead of "docs" put "tablesaw" and you should be good. You can verify by changing the url and pasting it into a browser.

Thanks again

**Reply by jbsooter** [reply]
Great! Thanks for taking a look. I adjusted the links, tested them, and submitted a pull request to resolve the issue.

**Comment by lwhite1** [comment]
Hi @jbsooter,
I will look into this and get back to you.

**Comment by lwhite1** [comment]
Hi @jbsooter,

That sounds great.  Let me know if you run into any issues.
